src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -208,7 +209,13 @@
     <ItemGroup>
       <_filesToCrossGen Include="@(FilesToPackage)" 
                         Condition="'%(FilesToPackage.IsNative)' != 'true' AND '%(FileName)' != 'System.Private.CoreLib' AND '%(FileName)' != 'mscorlib' AND '%(Extension)' == '.dll'">
+        <CrossGenedDirectory>$(CrossGenOutputPath)/%(TargetPath)/</CrossGenedDirectory>
         <CrossGenedPath>$(CrossGenOutputPath)/%(TargetPath)/%(FileName)%(Extension)</CrossGenedPath>
+        
+        <!-- Note on non-Windows this path isn't accurate because crossgen inserts the hash of the crossgen'd image
+             into the symbol's file name. So incremental build won't work correctly on non-Windows.
```

Comment:
@eerhardt, you're correct, the file names need to remain as they are set with GUIDs.  The GUID is used to map back to the native image so that we can validate that we have the right symbols.